creator-node/src/routes/files.js

```diff
@@ -303,6 +303,20 @@ const getCID = async (req, res) => {
             `${logPrefix} No valid file found for provided CID`
           )
         )
+      } else if (queryResults.type === 'dir') {
```

Comment:
this probably would be a bit cleaner with just 

if ( ... ) {
  return
}

if ( ... ) {
  return
}

decisionTree.push({DB_CID_QUERY_CIDFOUND})